OpenXmlPowerTools/PegBase.cs

```diff
@@ -896,55 +1163,76 @@ public BytesetData(Range[] r, byte[] c, bool bNegated)
                      foreach (int val in c) charSet_[val] = true;
                  }
 
-
                  public bool Matches(byte c)
                  {
                      bool bMatches = c < charSet_.Length && charSet_[(int)c];
                      if (bNegated_) return !bMatches;
                      else return bMatches;
                  }
              }*/
+
         #endregion PEG optimizations
+
         #region Constructors
+
         public PegByteParser()
             : this(null)
         {
         }
-        public PegByteParser(byte[] src):base(null)
+
+        public PegByteParser(byte[] src) : base(null)
         {
             SetSource(src);
         }
-        public PegByteParser(byte[] src, TextWriter errOut):base(errOut)
+
+        public PegByteParser(byte[] src, TextWriter errOut) : base(errOut)
         {
             SetSource(src);
         }
+
         #endregion Constructors
+
         #region Reinitialization, Source Code access, TextWriter access,Tree Access
+
         public void Construct(byte[] src, TextWriter Fout)
         {
             base.Construct(Fout);
             SetSource(src);
         }
+
         public void SetSource(byte[] src)
         {
-            if (src == null) src = new byte[0];
+            if (src == null)
+            {
+                src = new byte[0];
+            }
+
             src_ = src; srcLen_ = src.Length;
-            errors.lineStarts = new SortedList<int, int>();
-            errors.lineStarts[0] = 1;
+            errors.lineStarts = new SortedList<int, int>
+            {
+                [0] = 1
+            };
         }
-        public byte[] GetSource() { return src_; }
-        
+
+        public byte[] GetSource()
+        {
+            return src_;
+        }
+
         #endregion Reinitialization, Source Code access, TextWriter access,Tree Access
+
         #region Setting host variables
-        public bool Into(Matcher toMatch,out byte[] into)
+
+        public bool Into(Matcher toMatch, out byte[] into)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename the parameter 'into' so that it does not duplicate the method name.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)